controllers/gce/controller/controller.go

```diff
@@ -306,6 +306,13 @@ func (lbc *LoadBalancerController) sync(key string) (err error) {
 		glog.V(3).Infof("Finished syncing %v", key)
 	}()
 
+	if ingExists {
+		ing := obj.(*extensions.Ingress)
+		if isGCEMultiClusterIngress(ing) {
+			return lbc.syncMultiClusterIngress(ing, nodeNames)
```

Comment:
current me if I am wrong, if an ingress is updated from `gce` to `gce-multi`. The rest of GCP resources will stay around?

Reply:
`lbc.CloudClusterManager.GC` above in line 303 will still be called which should delete those resources.

Still, that is an interesting case I hadnt thought of. What if we say that users can not "upgrade" an existing gce ingress to gce-multi. They will always have to delete the existing gce ingress and create a new gce-multi ingress?